content/en/docs/application-store/built-in-apps/rabbitmq-app.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Deploy RabbitMQ"
+keywords: 'demploy, RabbitMQ'
+description: 'Deploy RabbitMQ'
+
+linkTitle: "Deploy RabbitMQ"
+weight: 2110
+---
+
+This tutorial walks you through an example of how to demploy RabbitMQ.
```

Comment:
You would better to add a simple introduction to RabbitMQ and add its official website link to RabbitMQ at the beginning.